lib/observer/src/observer.erl

```diff
@@ -19,11 +19,35 @@
 
 -module(observer).
 
--export([start/0, stop/0]).
+-export([start/0, start/1, start_and_wait/1, stop/0]).
 
 
 start() ->
     observer_wx:start().
 
+start(Node) when is_atom(Node) ->
+    start([Node]);
+start([Node]) ->
+    Node1 = to_atom(Node),
+    net_adm:ping(Node1),
+    Res = observer_wx:start(),
+    observer_wx:set_node(Node1),
+    Res.
+
+start_and_wait(Node) when is_atom(Node) ->
+    start_and_wait([Node]);
+start_and_wait([Node]) ->
+    start(Node),
```

Comment:
Check return value here

Reply:
Is a straight match (like it's done now) enough or would you want a proper error?

Reply:
I think it would be nicer to 'case of' it and return the error to the user.